lib/schema/schemaEnv.js

```diff
@@ -18,17 +18,28 @@
 'use strict';
 
 var config = require('amoeba').config;
-var schema = require('./schema.js');
+var misc = require('../misc.js');
+
+function usePlatform(userId) {
```

Comment:
Is this going to reload the user ids from the file every time the `info` API is called? If so, consider loading the hashed user id array once upon initialization and then simply checking against that pre-loaded hashed user id array.